packages/platform-bun/tsconfig.src.json

```diff
@@ -6,6 +6,6 @@
     "tsBuildInfoFile": ".tsbuildinfo/src.tsbuildinfo",
     "rootDir": "src",
     "outDir": "build/src",
-    "types": ["bun-types"]
+    "types": ["bun-types", "node"]
```

Comment:
We should remove this again when possible.

Reply:
Done, it was not needed anyway